extensions/rich_text_components/components_test.py

```diff
@@ -217,6 +217,29 @@ def test_video_validation(self):
         self.check_validation(
             components.Video, valid_items, invalid_items)
 
+    def test_svg_editor_validation(self):
+        """Tests svg editor component validation."""
+        valid_items = [{
+            'svg_filepath-with-value': 'random.svg',
```

Comment:
is this going to be a file path or filename?

Reply:
It is going to be a filename which will be used to fetch the filepath.

Reply:
So this should be svg_filename isn't it?

Reply:
I have made the change. Done.